configs/mainnet.yaml

```diff
@@ -144,14 +144,26 @@ ATTESTATION_SUBNET_PREFIX_BITS: 6
 # Deneb
 # `2**7` (=128)
 MAX_REQUEST_BLOCKS_DENEB: 128
-# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
-MAX_REQUEST_BLOB_SIDECARS: 768
 # `2**12` (= 4096 epochs, ~18 days)
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
 BLOB_SIDECAR_SUBNET_COUNT: 6
-## `uint64(6)`
+# `uint64(6)`
 MAX_BLOBS_PER_BLOCK: 6
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
+MAX_REQUEST_BLOB_SIDECARS: 768
+
+# Electra
+# 2**7 * 10**9 (= 128,000,000,000)
+MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000
+# 2**8 * 10**9 (= 256,000,000,000)
+MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT: 256000000000
+# `9`
+BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: 9
```

Comment:
Is there a reason why we keep separate `BLOB_SIDECAR_SUBNET_COUNT_ELECTRA` and `MAX_BLOBS_PER_BLOCK_ELECTRA` params? IMU the subnet count with the current design (one blob per subnet) will always be `MAX_BLOBS_PER_BLOCK_ELECTRA`

Reply:
Yes there's one blob per subnet but it doesn't necessarily have to be.

https://github.com/ethereum/consensus-specs/blob/0d5ad32355437e00471592de0adf2d2163961f47/specs/deneb/validator.md?plain=1#L183-L184

If for whatever reason we wanted to change this, we could.